tests/foreman/ui/test_ansible.py

```diff
@@ -556,6 +556,87 @@ def test_positive_non_admin_user_access_with_usergroup(
             ansible_roles_table = session.host_new.get_ansible_roles(target_sat.hostname)
             assert ansible_roles_table[0]['Name'] == SELECTED_ROLE
 
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_list('8')
+    def test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors(
+        self,
+        rhel_contenthost,
+        module_target_sat,
+        module_org,
+        module_location,
+        module_activation_key,
+    ):
+        """Check that Ansible tasks that make changes on a host show as notice in the config report and
+        failed Ansible tasks show as errors in the config report
+
+        :id: 286048f8-0f4f-4a3c-b5c7-fe9c7af8a780
+
+        :steps:
+            1. Import Ansible Roles
+            2. Assign Ansible roles to a host
+            3. Run Ansible Roles on a host
+            4. Check Config Report
+
+        :expectedresults: Verify that any tasks that make changes on the host are listed as notice in the config reportCaseLevel and
+            Verify that any task failures are listed as errors in the config reportCaseLevel
+        """
+        module_target_sat.cli.Capsule.update(
+            {
+                'name': module_target_sat.hostname,
+                'location-ids': module_location.id,
+                'organization-ids': module_org.id,
+            }
+        )
+        SELECTED_ROLE = 'theforeman.foreman_scap_client'
+        id = module_target_sat.nailgun_smart_proxy.id
+        module_target_sat.api.AnsibleRoles().sync(
+            data={'proxy_id': id, 'role_names': SELECTED_ROLE}
+        )
+        rhel_contenthost.create_custom_repos(rhel8_baseos=settings.repos.rhel8_os.baseos)
+        rhel_contenthost.create_custom_repos(rhel8_appstream=settings.repos.rhel8_os.appstream)
+        result = rhel_contenthost.register(
+            module_org, module_location, module_activation_key.name, module_target_sat
+        )
+        assert result.status == 0, f'Failed to register host: {result.stderr}'
+        with module_target_sat.ui_session() as session:
+            session.location.select(module_location.name)
+            session.organization.select(module_org.name)
+            session.host_new.add_single_ansible_role(rhel_contenthost.hostname)
+            ansible_roles_table = session.host_new.get_ansible_roles(rhel_contenthost.hostname)
+            assert ansible_roles_table[0]['Name'] == SELECTED_ROLE
+            # check err log for config report after ansible job scheduled
+            session.host_new.run_job(rhel_contenthost.hostname)
+            session.jobinvocation.wait_job_invocation_state(
+                entity_name='Run ansible roles',
+                host_name=rhel_contenthost.hostname,
+                expected_state='failed',
+            )
+            err_log = session.configreport.search(rhel_contenthost.hostname)
+            ROLE = SELECTED_ROLE.split('.')[1]
```

Comment:
foreman_scap_client is not role, its a package name
```suggestion
            package_name = SELECTED_ROLE.split('.')[1]
```

Reply:
updated object name